bin/navsnmp

```diff
@@ -86,7 +87,12 @@ def snmp_printer(netbox):
     print("# {}".format(netbox.sysname), file=sys.stderr)
     print(
         "-v{version} -c {community} {ipaddr}".format(
-            version=version, community=netbox.read_only, ipaddr=ipaddr
+            version=version,
+            community=getattr(
+                netbox.get_preferred_snmp_management_profile(writable=False),
+                "snmp_community",
+            ),
```

Comment:
This seems kind of odd. why are you using `getattr` to get the attribute?

Also, you already have the `profile` variable. To me, this seems more reasonable:

```suggestion
            community=profile.snmp_community,
```